submit-api/src/submit_api/models/role.py

```diff
@@ -0,0 +1,19 @@
+"""Role model class.
+
+Manages the role
+"""
+from __future__ import annotations
+
+from sqlalchemy import Column
+
+from .base_model import BaseModel
+from .db import db
+
+
+class Role(BaseModel):
+    """Definition of the Role entity."""
+
+    __tablename__ = 'role'
+
+    id = Column(db.Integer, primary_key=True, autoincrement=True)
+    role_name = Column(db.String(50), nullable=False)
```

Comment:
whats the role like ? not the keycloak role rite? can you add more comments here plz...